src/queue/song.py

```diff
@@ -0,0 +1,42 @@
+class Song:
+
+    def __init__(self, code, votes, ip):
+        self.code = code
+        self.votes = votes
```

Comment:
This might be redundant. We can get the votes by counting the number upvotes and downvotes

Reply:
Faster access speeds for low iterative cost.  I think it's beneficial to include.